src/makeCalculator.js

```diff
@@ -38,6 +38,40 @@
  */
 function makeCalculator() {
   // write code here
+
+  const calculator = {
```

Comment:
Don't leave unnecessary comments
```suggestion
function makeCalculator() {
  const calculator = {
```